src/algorithms/truncation/bond_truncation.jl

```diff
@@ -15,22 +15,23 @@ The truncation algorithm can be constructed from the following keyword arguments
 
 * `trunc::TruncationStrategy`: SVD truncation strategy when initilizing the truncated tensors connected by the bond.
 * `maxiter::Int=50` : Maximal number of ALS iterations.
-* `tol::Float64=1e-15` : ALS converges when fidelity change between two iterations is smaller than `tol`.
+* `tol::Float64=1e-9` : ALS converges when the change in bond SVD spectrum (normalized by maximum element) between two iterations is smaller than `tol`.
```

Comment:
```suggestion
* `tol::Float64=1e-9` : ALS converges when the relative change in bond SVD spectrum between two iterations is smaller than `tol`.
```

This might read slightly easier?